src/pages/console/uibuilder/databinding.mdx

```diff
@@ -3,7 +3,7 @@ export const meta = {
   description: `Figma to React code with Amplify Studio`,
 };
 
-Amplify Studio offers visual data binding capabilities to bind UI components to data to build dynamic components. Use the UI component editor to map fields to static values, or map fields to your backend data model. All data bindings get automatically included in generated code.
+[Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app) offers visual data binding capabilities to bind UI components to data to build dynamic components. Use the UI component editor to map fields to static values, or map fields to your backend data model. All data bindings get automatically included in generated code.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside Studio